packages/components/src/DateTimePickers/DateTimePicker/DateTimePicker.component.js

```diff
@@ -134,36 +148,65 @@ class DateTimePicker extends React.Component {
 		if (this.state.isDateTimeView) {
 			viewElement = (
 				<DateTimeView
-					onClickTitle={this.setMonthYearView}
+					allowFocus={this.state.allowFocus}
 					calendar={this.state.calendar}
-					onSelectMonthYear={this.onSelectCalendarMonthYear}
 					onSelectDate={this.onSelectDate}
-					selectedDate={this.state.selectedDate}
+					onSelectMonthYear={this.onSelectCalendarMonthYear}
 					onSelectTime={this.onSelectTime}
+					onTitleClick={this.setMonthYearView}
+					selectedDate={this.state.selectedDate}
 					selectedTime={this.state.selectedTime}
 				/>
 			);
 		} else {
 			viewElement = (
 				<MonthYearView
-					onClickBack={this.setDateTimeView}
-					selectedMonthIndex={this.state.calendar.monthIndex}
-					selectedYear={this.state.calendar.year}
+					allowFocus={this.state.allowFocus}
+					onBackClick={this.setDateTimeView}
 					onSelectMonth={this.onSelectCalendarMonth}
 					onSelectYear={this.onSelectCalendarYear}
+					selectedMonthIndex={this.state.calendar.monthIndex}
+					selectedYear={this.state.calendar.year}
 				/>
 			);
 		}
 
-		return <div className={theme.container}>{viewElement}</div>;
+		return (
+			<div
+				className={theme.container}
+				ref={ref => {
+					this.ref = ref;
+				}}
```

Comment:
Since there are multiple things to change, no need to mark only this one. We will need to check the entire codebase.
What do you propose as the new name ? Because it's on the container div, but don't really like the `container` word that means a lot of things.